tests/model_fields/test_uuid.py

```diff
@@ -153,7 +153,7 @@ def test_underlying_field(self):
         self.assertEqual(related.uuid_fk.pk, related.uuid_fk_id)
 
     def test_update_with_related_model_instance(self):
-        # regression for #24611
+        # (#24611).
```

Comment:
Adding a useful comment for the comments that are only "# regression for #24611" might be out of the scope for this PR but I don't think the change of putting the ticket number followed by a period is an improvement for these cases.